### PR TITLE
Feat/add architecture diagram

### PR DESCRIPTION
**Pull request by CastaChick**
ref: #144

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ Supports both fenced markdown (see screenshots), and mmd files.
 
 <img src="https://raw.githubusercontent.com/bpruitt-goddard/vscode-mermaid-syntax-highlight/master/images/quadrantChart.png" alt="Quadrant Chart Screenshot" width="325">
 
+### Architecture Diagram
+
+<img src="https://raw.githubusercontent.com/castachick/vscode-mermaid-syntax-highlight/master/images/architecture.png" alt="Architecture Diagram Screenshot" width="500">
```
Comment (bpruitt-goddard) [inline README.md:57]: For consistency, can this up be updated to pull from this repo?

**Comment by bpruitt-goddard** [comment]
If we want to bump the version and deploy with this PR merge, can you add two extra changes? I would bump the minor version to `1.7.0` as I've been increasing minor with changes like new diagram types.
1. Bump the version in [package.json](https://github.com/bpruitt-goddard/vscode-mermaid-syntax-highlight/blob/master/package.json#L6C1-L7C1)
2. Add the new version to the top of the [changelog](https://github.com/bpruitt-goddard/vscode-mermaid-syntax-highlight/blob/master/CHANGELOG.md)
     - _Note_ There is a copy/paste error in the last update so the most recent line should really be `1.6.7` and this change should be `1.7.0`.

**Review by bpruitt-goddard — APPROVED** [review]

**Comment by CastaChick** [comment]
@bpruitt-goddard Thank you for your review. I've bumped version to v1.7.0.

**Comment by koshea** [comment]
I tested this as well, works great, thank you.

**Pull request merged**